src/pear/config.vsh.yaml

```diff
@@ -0,0 +1,166 @@
+            Specify the quality threshold for trimming the low quality part of a read. If the quality scores of two consecutive bases are strictly less than the specified threshold, the rest of the read will be trimmed.
+          required: false
+          example: 0
+        - name: --max_uncalled_base
+          alternatives: -u
+          type: double
+          description: |
+            Specify the maximal proportion of uncalled bases in a read. Setting this value to 0 will cause PEAR to discard all reads containing uncalled bases. The other extreme setting is 1 which causes PEAR to process all reads independent on the number of uncalled bases.
+          example: 1.0
+          required: false
+        - name: --test_method
+          alternatives: -g
+          type: integer
+          description: |
+            Specify the type of statistical test. Two options are available. 1: Given the minimum allowed overlap, test using the highest OES. Note that due to its discrete nature, this test usually yields a lower p-value for the assembled read than the cut- off (specified by -p). For example, setting the cut-off to 0.05 using this test, the assembled reads might have an actual p-value of 0.02.
+            2. Use the acceptance probability (m.a.p). This test methods computes the same probability as test method 1. However, it assumes that the minimal overlap is the observed overlap with the highest OES, instead of the one specified by -v. Therefore, this is not a valid statistical test and the 'p-value' is in fact the maximal probability for accepting the assembly. Nevertheless, we observed in practice that for the case the actual overlap sizes are relatively small, test 2 can correctly assemble more reads with only slightly higher false-positive rate.
+          required: false
+          example: 1
+        - name: --emperical_freqs
+          alternatives: -e
+          type: boolean_true
+          description: |
+            Disable empirical base frequencies.
+        - name: --score_method
+          alternatives: -s
+          type: integer
+          description: |
+            Specify the scoring method. 1. OES with +1 for match and -1 for mismatch. 2: Assembly score (AS). Use +1 for match and -1 for mismatch multiplied by base quality scores. 3: Ignore quality scores and use +1 for a match and -1 for a mismatch.
+          required: false
+          example: 2
+        - name: --phred_base
+          alternatives: -b
+          type: integer
+          description: |
+            Base PHRED quality score.
+          required: false
+          example: 33
+        - name: --memory
+          alternatives: -y
+          type: string
+          description: |
+            Specify the amount of memory to be used. The number may be followed by one of the letters K, M, or G denoting Kilobytes, Megabytes and Gigabytes, respectively. Bytes are assumed in case no letter is specified.
+          required: false
+          example: 4G
```

Comment:
Use `$meta_memory_gb` for this ;)

```suggestion
```

Reply:
Pear memory argument only supports b,k,m,g

Can I also use this with viash memory or does viash automatically convert this to bytes?

Reply:
With Viash, you can pass a value to `---memory` in different units -- B, KB, MB, GB, TB. Viash will then convert any value into all possible units: `meta_memory_b`, `meta_memory_kb`, `meta_memory_mb`. That is, if you specify `---memory 1TB`, all possible `meta_memory_*` fields will still be created with the respective integer/long values inside.